### PR TITLE
Dsn 14 index page home

### PR DESCRIPTION
**Pull request by yamenchaban**
Dsn-14 index page home

### CODE REVIEW

**Comment by patvoj** [comment]
I can see old commits from your previous task in this branch, so I think you mistakenly created this branch out of your previous branch instead of updated Develop branch. Or maybe from Develop branch, but not updated

**Review by bela-riboczki-ext — CHANGES_REQUESTED** [review]
As we talked about it, please add this page as a frontend component.

File: DSN/src/main/resources/images/74130270.png
Comment (bela-riboczki-ext) [inline DSN/src/main/resources/images/74130270.png:1]: Can you rename this file to something which has more meaning in the context of the application?
Thank you.

File: DSN/frontend/src/app/page.tsx [hunk 1]
```diff
@@ -1,113 +1,281 @@
-import Image from "next/image";
+import React from 'react';
+import Image from 'next/image'; // استيراد مكون Image من Next.js
+import profileImage from '../../public/images/74130270.png'; // استيراد الصورة
+import '../styles/Index_Page_Home_style.css';
 
-export default function Home() {
+const IndexPage: React.FC = () => {
```
Comment (bela-riboczki-ext) [inline DSN/frontend/src/app/page.tsx]: I think you can export directly this as the content, there is no need to define as a constant here.
Like how it was before in the tutorial's code:
`export default function IndexPage() { * Page contents * }`

**Review by bela-riboczki-ext — COMMENTED** [review]
Please review the comments.

File: DSN/src/main/resources/images/74130270.png
Comment (yamenchaban) [reply DSN/src/main/resources/images/74130270.png:1]: Done
Thank you for your note

File: DSN/frontend/src/app/page.tsx (hunk 1 above)
Comment (yamenchaban) [reply DSN/frontend/src/app/page.tsx]: Done
Thank you for your note

**Review by bela-riboczki-ext — APPROVED** [review]
Looks good from my side.

File: DSN/frontend/src/app/page.tsx
Comment (patvoj) [inline DSN/frontend/src/app/page.tsx:1]: Could you maybe move all files inside a folder beneath app directory? So in this case the root would be something like `src/app/index`. NextJS has automatical routing, so if you have directory structure like this, then you should be able easily open your page by just visiting localhost:port/index
image

NextJS routing: https://nextjs.org/docs/app/building-your-application/routing

Comment (patvoj) [reply DSN/frontend/src/app/page.tsx:1]: hmm, on the other hand it makes sense to have the main page without anything after domain, so it's only http://localhost:3000 and nothing after that. @bela-riboczki-ext  What do you think?

Besides that I like the old facebook design, and I can see there is already used some area box for posts, so we could maybe used that later

Comment (yamenchaban) [reply DSN/frontend/src/app/page.tsx:1]: It is a Home Page that should remain without a folder

Comment (patvoj) [reply DSN/frontend/src/app/page.tsx:1]: Yeah, I realised it's actually correct. So it's alright from my side and I approve this PR

**Review by patvoj — APPROVED** [review]
All good from my side

**Pull request merged**